content/_code-samples/send-a-memo/js/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "js",
+  "version": "1.0.0",
+  "description": "",
+  "main": "send-a-memo.js",
+  "scripts": {
+    "start": "node send-a-memo.js"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Technically, the dev portal's original contents are offered under the MIT, not ISC license. (I think we may switch to some Creative Commons 4 license which is more appropriate for documentation.)

For very basic code samples like this, I think it's more appropriate to declare them public domain—under many jurisdictions, copying a sample like this would fall under Fair Use anyway. The **CC0-1.0** license declares something to be public domain, so that would remove ambiguity over that.

```suggestion
  "license": "CC0-1.0",
```